### DISCUSSION TITLE
Model size increases after download from editor

### DISCUSSION THREAD

**Discussion by pegot** [google/model-viewer#4304] [Q&A]
I uploaded to the editor a Blender compressed .glb file that was 2.7mb.  But after download the package from the ModelViwer editor. the file size increased to 10mb.

What is ModelViewer doing or adding to the model to increase its size?

**Comment by elalish** [comment]
Hmm, great question. We fixed a bug related to this some time ago - perhaps there's another version of it? Can you link the original model so we can repro? 

I believe the problem is that when three.js exports the scene it uses the browser to re-encode the textures, since it doesn't keep the original compressed ones. Generally it does a pretty good job, but it's browser-specific (is this on Chrome?). Honestly, if you use Blender, that's a much more capable editing platform that our editor, which is mostly for scene setup.

**Comment by pegot** [comment]
I'm new to model-viewer so initially using the editor to set everything up.  The compressed Blender .glb that the editor is increasing file size of after downloading is attached.  Originally 2.7 mb but 10 mb after downloading from the editor.

[NTI-Statue-reduced-cleaned-BackfaceCulling.zip](https://github.com/google/model-viewer/files/11641689/NTI-Statue-reduced-cleaned-BackfaceCulling.zip)

**Reply by elalish** [reply]
Thanks, I'll look into it. Just replace it with your original GLB and everything should work fine.
